project.mml

```diff
@@ -139,6 +139,7 @@ Layer:
               ('railway_' || (CASE WHEN railway = 'station' THEN railway ELSE NULL END)) AS railway,
               CASE WHEN religion IN ('christian', 'jewish', 'muslim') THEN religion ELSE 'INT-generic'::text END AS religion,
               way_area/NULLIF(!pixel_width!::real*!pixel_height!::real,0) AS way_pixels,
+              CASE WHEN building = 'no' OR building IS NULL THEN 'no' ELSE 'yes' END AS is_building -- always no with the where conditions
```

Comment:
Comma is missing before the comment, so it does not compile, it should be:

```sql               
CASE WHEN building = 'no' OR building IS NULL THEN 'no' ELSE 'yes' END AS is_building, -- always no with the where conditions 
```

Reply:
strange, it works fine without and I get a syntax error in Postgis when I add the comma.

Reply:
Very strange  indeed... Could anyone else check it? @Adamant36, @jeisenbe?

Reply:
@matkoniecz @sommerluk Maybe you could check it too?

Reply:
I resolved the problem